src/main/java/teammates/ui/controller/InstructorFeedbackResultsDownloadAction.java

```diff
@@ -37,8 +37,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 statusToAdmin = "Summary data for Feedback Session " + feedbackSessionName
                               + " in Course " + courseId + " was downloaded";
             } else {
-                fileContent = logic.getFeedbackSessionResultSummaryInSectionAsCsv(courseId, feedbackSessionName,
-                                                                                  instructor.email, section);
+                fileContent =
+                        logic.getFeedbackSessionResultSummaryInSectionAsCsv(
+                                courseId, feedbackSessionName, instructor.email, section);
```

Comment:
Maybe keep `logic.getFeedback...` on the first line, and indent the second line with respect to `[l]ogic`